src/main/java/com/google/api/codegen/discogapic/transformer/java/JavaDiscoGapicRequestToViewTransformer.java

```diff
@@ -239,42 +243,44 @@ private StaticLangApiMessageView generateRequestClass(
     }
 
     for (FieldModel entry : method.getInputFields()) {
-      properties.add(schemaToParamView(context, entry, symbolTable));
+      properties.add(schemaToParamView(context, entry, entry.getSimpleName(), symbolTable, true));
```

Comment:
I can't tell what the `true` means from the calling site here. I recommend creating an enum for the parameter.

Reply:
Done.